workloads/router-perf-v2/templates/nginx-pod.yml

```diff
@@ -1,18 +1,25 @@
-apiVersion: v1
-kind: Pod
+apiVersion: apps/v1
```

Comment:
`templates/nginx-deploy.yml` already exists which is a deployment of nginx pods.